final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,76 @@
+import feedparser
+from pprint import pprint
+from arg import parsargs, vers
+from clean_output import clean_title
+from bs4 import BeautifulSoup 
+import json
+from loggs import logg, logg_json
+import logging
+
+args = parsargs()
+
+parsed = feedparser.parse(args.source)
```

Comment:
It is better to avoid using global variables whenever you can. For example here you can pass args as an argument to a function if you need,